packages/big-design/src/components/Form/Label/styled.tsx

```diff
@@ -19,12 +19,12 @@ export const StyledLabel = styled<
   display: inline-block;
   margin-bottom: ${({ theme }: StyledLabelArgument) => theme.spacing.xxSmall};
 
-  ${({ theme, renderOptional }: StyledLabelArgument) =>
+  ${({ theme, renderOptional, localization }: StyledLabelArgument) =>
     renderOptional &&
     css`
       &::after {
         color: ${theme.colors.secondary60};
-        content: ' (optional)';
+        content: ' (${localization?.optional})';
```

Comment:
🤔 if `localization` is nullable, that means `localization?.optional` can be `undefined`. Are types correct?